doc/ref_cert/RC1/chapters/chapter03.md

```diff
@@ -67,7 +67,7 @@ For detailed information on RM & RA-1 NFVI and VNF requirements, please refer to
 |9|req.inf.com.01|must provide compute resources for VM instances.|Infrastructure|in Functest|Must|Functional||Functest||
 |10|req.inf.com.02|should include industry standard hardware management systems at both HW device and platform level|Infrastructure - baremental testing|missing in Functest. Captured in Telco TCs Baremetal - validations.||||||
 |11|req.inf.com.03|should support symmetrical CPU multi-processing with shared memory access as well as multi-threading.|Infrastructure|in Functest||||Functest||
-|12|req.inf.com.04|must be able to support multiple CPU SKU options to support Base, Network Intensive, and Compute Intensive infrastructure profiles.|Infrastructure||Must|NFR||||
+|12|req.inf.com.04|must be able to support multiple CPU types to support Base, Network Intensive, and Compute Intensive infrastructure profiles.|Infrastructure||Must|NFR||||
```

Comment:
@EditKoselak Can you define "CPU Types" (it may have to be changed in RA-1 Ch02 first) as there is no agreed to meaning for the term? The definition can be added as a "Note: ...." after the end of the sentence.

Reply:
A definition of "CPU types" could be the following:
CPUs that support the same features needed for the execution of computer programs (for example, instruction sets, cache size, number of cores).

What do you think about this definition? Should we add it to the glossary as well?

Reply:
@EditKoselak Yes, we should add to Glossary. Some minor suggestions:

CPU Types: is a classification of CPUs by features needed for the execution of computer programs; for example, instruction sets, cache size, number of cores.

Reply:
@pgoyal01, we have added the term to the glossary. Is that enough or should we still add the note with the definition here as well?

Reply:
@EditKoselak Thanks.